PC/VS9.0/pyproject.vsprops

```diff
@@ -12,7 +12,7 @@
 		InlineFunctionExpansion="1"
 		EnableIntrinsicFunctions="true"
 		AdditionalIncludeDirectories="..\..\Include; ..\..\PC"
-		PreprocessorDefinitions="_WIN32"
+		PreprocessorDefinitions="_WIN32;WIN32"
```

Comment:
Ok fine. I'm just surprised that we have to define it manualy. It should already be defined by <windows.h> no?